includes/features/drupal.org/xml_sitemap/xml_sitemap.feature

```diff
@@ -1,9 +1,16 @@
 Feature: XML_sitemap
   Verify XML_SiteMap functionality
 
-  @api
+  @api @live
   Scenario: XML Sitemap
     Given I am logged in as a user with the "administrator" role
     And I run cron
     When I go to "/sitemap.xml"
-    Then I should get a "200" HTTP response
\ No newline at end of file
+    Then I should get a "200" HTTP response
```

Comment:
Yup. One for when the site is in development @dev, and one is when the site is live @live.